packit/api.py

```diff
@@ -150,13 +151,32 @@ def sync_release(
         :param dist_git_branch: branch in dist-git
         :param use_local_content: don't check out anything
         :param version: upstream version to update in Fedora
+        :param tag: upstream git tag
         :param force_new_sources: don't check the lookaside cache and perform new-sources
         :param upstream_ref: for a source-git repo, use this ref as the latest upstream commit
         :param create_pr: create a pull request if set to True
         :param force: ignore changes in the git index
 
         :return created PullRequest if create_pr is True, else None
         """
+
+        # process version and tag parameters
+        if version and tag:
+            raise PackitException(
+                "Function parameters version and tag are mutually exclusive."
+            )
+        elif not tag:
+            if not version:
+                version = self.up.get_version()
+                if not version:
+                    raise PackitException(
+                        "Could not figure out version of latest upstream release."
+                    )
+            upstream_tag = self.up.convert_version_to_tag(version)
```

Comment:
Do I understand correctly that supporting this is needed, because on the CLI `packit propose-update` requires a version to be provided, while in Packit Service the webhooks deliver a Git tag?

Reply:
I think so.

Reply:
yes